### PR TITLE
Relax the restrictions on rubocop to allow for upgrading to > rubocop…

### PR DESCRIPTION
**Pull request by knappe**
… 1.0.

So this is quite gross, but the tests pass.  I think we need to decide if the tests are worth keeping though.

### CODE REVIEW

File: pronto-rubocop.gemspec [hunk 1]
```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.summary = 'Pronto runner for Rubocop, ruby code analyzer'
 
   s.licenses = ['MIT']
-  s.required_ruby_version = '>= 2.3.0'
+  s.required_ruby_version = '>= 2.4.0'
```
Comment (knappe) [inline pronto-rubocop.gemspec:17]: Works around https://github.com/rubocop-hq/rubocop/issues/9482

**Comment by knappe** [comment]
For reference: https://github.com/prontolabs/pronto-rubocop/pull/56#issuecomment-770462395

File: pronto-rubocop.gemspec (hunk 1 above)
Comment (ashkulz) [reply pronto-rubocop.gemspec:17]: This breaks the CI, you should probably exclude Ruby 2.3 [like done for pronto-haml](https://github.com/prontolabs/pronto-haml/blob/8e230f3/.github/workflows/checks.yml#L15).

**Review by ashkulz — COMMENTED** [review]
Thanks for the attempt! Not sure if you saw [this comment](https://github.com/prontolabs/pronto-rubocop/pull/60#issuecomment-773779213), but I _think_ this PR addresses points 1 (after you skip Ruby 2.3) & 2. I think we still need to explicitly test against different RuboCop versions via CI, though.

**Comment by ashkulz** [comment]
I think #60 is much closer to merging as tests pass on that PR, and I also fixed compatibility with older RuboCop versions on `master` which was discovered there.

**Comment by ashkulz** [comment]
I'm planning to merge the other PR and cut a release this weekend. Thanks for the contribution anyway 🎉

**Comment by knappe** [comment]
@ashkulz Not a problem.  I'm glad to see this kept moving in #60. 😄

**Comment by ashkulz** [comment]
pronto-rubocop 0.11.1 has been released, which supports RuboCop >= 1.0 🎉

**Pull request closed without merging**